Dockerfile

```diff
@@ -37,7 +39,7 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
         docker-ce=$DOCKER_VERSION \
         docker-ce-cli=$DOCKER_VERSION \
         containerd.io=$CONTAINERD_VERSION
-
+		
 # Install Sonar Scanner
 RUN apt-get install -y unzip \
```

Comment:
Changing the meta-information tags should be reflected in the rest of the file:
`NETCOREAPP_VERSION` to `NETAPP_VERSION`

Reply:
Done